e2e-tests/content-strings/fields/insurance/your-buyer/index.js

```diff
@@ -34,8 +34,8 @@ export const YOUR_BUYER_FIELDS = {
       },
     },
     [COMPANY_OR_ORGANISATION.ADDRESS]: {
-      LABEL: 'Address',
-      MAXIMUM: 330,
+      LABEL: 'Company address',
+      MAXIMUM: 1000,
```

Comment:
Does the keystone schema and DB dump have the same `varchar` for this?

Reply:
yes that is set to 1000